src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -4686,3 +4686,132 @@ def test_ps_s3_notification_push_kafka_security_ssl_sasl_scram():
 def test_ps_s3_notification_push_kafka_security_sasl_scram():
     kafka_security('SASL_PLAINTEXT', mechanism='SCRAM-SHA-256')
 
+
+@attr('data_path_v2_test')
+def test_ps_s3_data_path_v2_migration():
+    """ test data path v2 migration """
+    conn = connection()
+    zonegroup = get_config_zonegroup()
+
+    # create random port for the http server
+    host = get_ip()
+    http_port = random.randint(10000, 20000)
+
+    # disable v2 notification
+    result = admin(['-c', 'run/c1/ceph.conf', 'zonegroup', 'modify', '--disable-feature=notification_v2'], get_config_cluster())
```

Comment:
```suggestion
    result = admin(['zonegroup', 'modify', '--disable-feature=notification_v2'], get_config_cluster())
```

it shoudl work without that. also, you should make assumption on the directory from which the test is being executed